### PR TITLE
Tibber component

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**  https://github.com/home-assistant/home-assistant/pull/17062

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/notify.tibber.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: "Tibber"
+description: "Instructions on how to integrate Tibber within Home Assistant."
+date: 2015-10-04 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tibber.png
+ha_category: Notifications
+ha_release: 0.81
```
Comment (frenck) [inline source/_components/notify.tibber.markdown]: "0.80" Right?

File: source/_components/tibber.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title: "Tibber"
+description: "Instructions on how to integrate Tibber within Home Assistant."
+date: 2015-10-04 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tibber.png
+ha_category: Energy
+ha_release: 0.81
```
Comment (frenck) [inline source/_components/tibber.markdown]: Same... "0.80"?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/notify.tibber.markdown (hunk 1 above)
Comment (Danielhiversen) [reply source/_components/notify.tibber.markdown]: Yes.
 I thought the component would not be merged before 0.81

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks @Danielhiversen 👍 

✅  Can be merged as soon as the build passes.

**Pull request merged**